app/scripts/background.js

```diff
@@ -1138,7 +1138,9 @@ export function setupController(
 
       connectEip1193(portStream, remotePort.sender);
 
-      if (isFirefox) {
+      // for firefox and manifest v2 (non production webpack builds)
+      // we expose the multichain provider via window.postMessage
+      if (isFirefox || !isManifestV3) {
```

Comment:
I've asked this before, but its still hard to tell....

Is it something about Firefox that doesn't support the alternative path, or only MV2 that needs to take this path?

In other words, if we did have an MV3 build that could run on Firefox, would we want to remove the `isFirefox` check here?

Reply:
Firefox does not currently support externally_connectable on MV2 or MV3.

Chromium MV2 has externally_connectable but without support for wildcarded domains, making it effectively impossible to use as an API entrypoint

Chromium MV3 supports both externally_connectable with wildcarded domains, making it actually possible to use as an API entrypoint.

So the working combo is supports externally_connectable with wildcarded domains.